Flexx/Classes/Flexx.swift

```diff
@@ -142,6 +142,16 @@ public class Flexx {
         return currentLocale
     }
     
+    /// Check locale identifier
+    /// - returns: Locale value with valid local identifier
+    public func checkLocaleId(localeId: String) -> String {
```

Comment:
Could you also pick a different name for the function. In my opinion `checkLocaleId` with return type `String` doesn't say much what to expect.

Reply:
done